### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by JoshK47YT** [Gucci637/Fentanyl#38]
![image](https://user-images.githubusercontent.com/86482377/162098092-101a1f0f-07b0-485e-a8c0-4f245b7253ff.png)

**Comment by JoshK47YT** [comment]
Still Does Not Work

**Comment by Z3leZ** [comment]
yeah i got the same problem but i guess he just didn't update the github info

**Comment by blooketpls** [comment]
same... cant work

**Comment by JoshK47YT** [comment]
> Should be fixed by now! Make sure you're using the latest version.

I am Using The Lastest Version and it is not working
I have Lost Trust in You Now Gonna Destroy This Project

**Comment by Z3leZ** [comment]
it sstill not working for me too i guess i ll do the same since i dont see anyone saying that it's working

**Comment by oqvh** [comment]
> it sstill not working for me too i guess i ll do the same since i dont see anyone saying that it's working

Because it doesnt fucking work hes just gonna try and stretch out this "project" for as long as he can fooling people
